test/test_cuda.py

```diff
@@ -1227,6 +1227,81 @@ def test_cat_autogpu(self):
         z = torch.cat([x, y], 0)
         self.assertEqual(z.get_device(), x.get_device())
 
+    def test_clamp_cuda(self):
+        m1 = torch.rand(100).mul(5).add(-2.5).cuda()  # uniform in [-2.5, 2.5]
+        # just in case we're extremely lucky.
+        min_val = -1
+        max_val = 1
+        m1[1] = min_val
+        m1[2] = max_val
+
+        res1 = m1.clone()
+        res1.clamp_(min_val, max_val)
+        res2 = m1.clone()
+        for i in iter_indices(res2):
```

Comment:
Is iter_indices imported / defined in this context? This seems to be the cause of the test failures.